server/util.py

```diff
@@ -148,7 +168,12 @@ def get_analysis_runner_metadata(
     Get well-formed analysis-runner metadata, requiring the core listed keys
     with some flexibility to provide your own keys (as **kwargs)
     """
-    output_dir = f'gs://cpg-{dataset}-{cpg_namespace(access_level)}/{output_prefix}'
+    if environment == 'gcp':
+        output_dir = f'gs://cpg-{dataset}-{cpg_namespace(access_level)}/{output_prefix}'
+    elif environment == 'azure':
+        # TODO: need a way for analysis runner to know where to save metadata
```

Comment:
It follows the same sort of convention right, where storage-account is `cpg{datasetWithoutTabs}`

```
azure://{storage-account}/{main,test}/{output_prefix}
```